src/Infrastructure/Symfony/Command/PurgeUserTokensCommand.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Symfony\Command;
+
+use App\Domain\User\Repository\TokenRepositoryInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand(
+    name: 'app:purge:user_tokens',
```

Comment:
(Détail de ché détail) J'ai remarqué que les commandes sont souvent avec le verbe à la fin ?

```suggestion
    name: 'app:user_tokens:purge',
```